spacepy/pybats/__init__.py

```diff
@@ -110,30 +110,60 @@ class methods with the *calc_* prefix and call them.
 
 '''
 
-__contact__ = 'Dan Welling, dwelling@umich.edu'
+__contact__ = 'Dan Welling, daniel.welling@uta.edu'
 
 # Global imports (used ubiquitously throughout this module.
 import os.path
+from functools import wraps
 
 from spacepy.datamodel import dmarray, SpaceData
-import spacepy.plot.apionly
-import spacepy.plot as spu
 import numpy as np
 
+# Pybats-related decorators:
+def calc_wrapper(meth):
+    '''
+    This is a decorator for `self.calc_*` object methods.  It establishes
+    `self._calcs`, a list of calcuation functions that have been called, and
+    adds the called function to that list to keep track of which calculations
+    have already been performed for the object.  If a calculation has already
+    been performed, it is skipped in order to reduce effort.
+    '''
+
```

Comment:
I know it's neither here nor there, but the whitespace lines surrounding the  `@wraps(meth)` make thing harder to read for me.

Reply:
Indeed; fixed.